samcli/commands/package/package_context.py

```diff
@@ -135,8 +136,12 @@ def run(self):
             raise PackageFailedError(template_file=self.template_file, ex=str(ex)) from ex
 
     def _export(self, template_path, use_json):
-        template = Template(template_path, os.getcwd(), self.uploaders, self.code_signer)
+        template = Template(template_path, os.getcwd(), self.uploaders, self.code_signer, normalize_template=True)
         exported_template = template.export()
+        updated_template = copy.deepcopy(exported_template)
+        ResourceMetadataNormalizer.normalize(updated_template, True)
```

Comment:
Where is `updated_template` is used? I saw it is been copied and send it to `ResourceMetadataNormalizer` but we are not using it afterwards.

Reply:
it is used to clone the normalized parameters to the exported template, any way I changed this code to remove the duplicate normalizing step.